doc-content/enterprise-only/openshift/rhpam73-authoring-ha-ref.adoc

```diff
@@ -334,15 +337,20 @@ Http Get on http://localhost:9200/_cluster/health
 |=======================================================================
 |Deployment |Variable name |Description |Example value
 
-.65+| `${APPLICATION_NAME}-rhpamcentr`
+.69+| `${APPLICATION_NAME}-rhpamcentr`
 |`KIE_ADMIN_USER` | KIE administrator username | `${KIE_ADMIN_USER}`
 |`KIE_ADMIN_PWD` | KIE administrator password | `${KIE_ADMIN_PWD}`
 |`KIE_MBEANS` | KIE server mbeans enabled/disabled (Sets the kie.mbeans and kie.scanner.mbeans system properties) | `${KIE_MBEANS}`
+|`KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE` | Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property) | `${KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE}`
+|`KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL` | KIE ServerTemplate Cache TTL in milliseconds (Sets the org.kie.server.controller.template.cache.ttl system property) | `${KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL}`
+|`KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED` | If set to true, turns on OpenShift integration feature (Sets the org.kie.workbench.controller.openshift.enabled system property) | `${KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED}`
 |`KIE_SERVER_CONTROLLER_USER` | KIE server controller username (Sets the org.kie.server.controller.user system property) | `${KIE_SERVER_CONTROLLER_USER}`
 |`KIE_SERVER_CONTROLLER_PWD` | KIE server controller password (Sets the org.kie.server.controller.pwd system property) | `${KIE_SERVER_CONTROLLER_PWD}`
 |`KIE_SERVER_CONTROLLER_TOKEN` | KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property) | `${KIE_SERVER_CONTROLLER_TOKEN}`
 |`KIE_SERVER_USER` | KIE server username (Sets the org.kie.server.user system property) | `${KIE_SERVER_USER}`
 |`KIE_SERVER_PWD` | KIE server password (Sets the org.kie.server.pwd system property) | `${KIE_SERVER_PWD}`
+|`WORKBENCH_ROUTE_NAME` | -- | `${APPLICATION_NAME}-rhpamcentr`
+|`MAVEN_MIRROR_URL` | Maven mirror to use by Business Central and KIE server. | `${MAVEN_MIRROR_URL}`
```

Comment:
s/Maven mirror to use by Business Central and KIE server/Maven mirror to use for Business Central and KIE server